.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: go
+
+go:
+ - 1.6
+ - tip
```

Comment:
I wonder if we'll want to support that. I imagine if 1.7 breaks all our code, we'll either stick with 1.6 or make 1.7 the new requirement and upgrade all the servers.